go.mod

```diff
@@ -1,10 +1,21 @@
 module github.com/networkservicemesh/api
 
-go 1.16
+go 1.18
 
 require (
 	github.com/pkg/errors v0.9.1
 	github.com/stretchr/testify v1.7.0
 	google.golang.org/grpc v1.32.0
 	google.golang.org/protobuf v1.23.0
 )
+
+require (
+	github.com/davecgh/go-spew v1.1.0 // indirect
+	github.com/golang/protobuf v1.4.0 // indirect
+	github.com/pmezard/go-difflib v1.0.0 // indirect
+	golang.org/x/net v0.0.0-20190311183353-d8887717615a // indirect
+	golang.org/x/sys v0.0.0-20190215142949-d0b11bdaac8a // indirect
+	golang.org/x/text v0.3.0 // indirect
+	google.golang.org/genproto v0.0.0-20190819201941-24fa4b261c55 // indirect
+	gopkg.in/yaml.v3 v3.0.0-20200313102051-9f266ea9e77c // indirect
```

Comment:
Note: go 1.17 the module graph had changed --> added  pruning/lazy loading.

See at  https://golang.org/doc/go1.17#go-command